spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/SkipperStreamService.java

```diff
@@ -243,7 +221,9 @@ public String convertPropertiesToSkipperYaml(String streamName, Map<String, Stri
 				appMap.put(SpringCloudDeployerApplicationManifest.SPEC_STRING, specMap);
 			}
 			if (appDeploymentRequest.getCommandlineArguments().size() == 1) {
-				appMap.put("version", appDeploymentRequest.getCommandlineArguments().get(0));
+				String version = appDeploymentRequest.getCommandlineArguments().get(0);
```

Comment:
Just a note to make the changes (either here or in the PR where we changed `spec.version`) to put `version` inside specMap.